crates/compiler/builtins/benchmark-dec.zig

```diff
@@ -129,8 +141,8 @@ fn avg_runs(comptime T: type, comptime n: usize, comptime op: fn (T, T) T, v: T)
         runs[i] = callWrapper(u64, .never_inline, run, .{ T, n, op, v });
     }
 
-    var real_runs = runs[warmups..runs.len];
-    std.sort.sort(u64, real_runs, {}, comptime std.sort.asc(u64));
+    const real_runs = runs[warmups..runs.len];
+    std.sort.insertion(u64, real_runs, {}, comptime std.sort.asc(u64));
```

Comment:
`std.sort.sort` is no longer available, I picked something that seemed reasonable. This doesn't appear to be run in any CI so it may not be correct.